index.html

```diff
@@ -6,47 +6,174 @@
+
+    <hr class="solid" />
+
+    <div class="languages"><img src="img_Rectangle.png" alt="rectangle">
+        Language
+        <ul>
+        <li>Javascript</li>
+        <li>Ruby</li>
+        <li>Html</li>
+        <li>Css</li>
+        </ul>
     </div>
 
-    <ul class=" social-media social-media-icons mv-social-media mv-social-media-icons">
-        <li class="social-1"><a href="#"><i class="fab fa-github fa-lg"></i></a></li>
-        <li class="social-2"><a href="#"><i class="fab fa-linkedin-in fa-lg"></i></a></li>
-        <li class="social-3"><a href="#"><i class="fab fa-angellist fa-lg"></i></a></li>
-        <li class="social-4"><a href="#"><i class="fab fa-twitter fa-lg"></i></a></li>
-        <li class="social-5"><a href="#"><i class="fab fa-medium-m"></i></a></li>
+    <div class="framework"><img src="img_Rectanglen.png" alt="">
+        Frameworks
+        <ul>
+        <li>Bootstraps</li>
+        <li>Ruby on Rails</li>
+        <li>RSPec</li>
+        <li>CapyBara</li>
+        <li>Selenium</li>
         </ul>
+    </div>
 
+    <div class="skill"><img src="img_Rectangles.png" alt="">
+        Skills
+        <ul>
+        <li>Codekit</li>
+        <li>Github</li>
+        <li>Codepen</li>
+        <li>Gitlab</li>
+        <li>Terminal</li>
+        </ul>
     </div>
     </section>
-</main>
+    <br>
+
+    <footer class="footer">
+    <h2 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in <span class="toll">touch.</span></h2>
+    
+    <form  class="player" action="https://formspree.io/f/xayagqrn" method="post">
+    <ul class="name">
+    <li class="form">
+    <label for="fullName">
+    <input type="text" name="full-name" placeholder="  Full name" maxlength="30" id="fullName" required>
+        </label></li>
+        
+    <li class="form">
+    <label for="email">
+        <input type="email" name="email" id="email" placeholder="  Email address" required>
+        </label></li>
+        
+        <li class="textare">
+        <label for="message">
+        <textarea name="message" id="message" cols="30" rows="6" maxlength="500" placeholder="Enter text here" required></textarea>
+        </label></li>
+    
+    <li class="buttonclub">
+    <button type="submit" class="button_class">Get in touch</button>
+    </li>
+    </ul>
+</form>
+
+    <div class="border"><img src="img_line-line.png" alt=""></div>
+
+    <ul class="icongroup">
+        <li class="githubx"><img src="pictures/githreal.svg" alt="Github"></li>
+
+        <li class="linkedinx"><img  src="Pictures/linkedreal.svg" alt="linkedin"></li>
+
+        <li class="angelistx"><img src="Pictures/angelist.svg" alt="angel.co"></li>
+
+        <li class="twitterx"><img src="Pictures/twittereal.svg" alt="twitter"></li>
+
+        <li class="mediumx"><img src="Pictures/mediumreal.svg" alt="medium.com"></li>
+    </ul>
+    <div class="shape">
+    <img src="img_Shape.png" alt="">
+    </div>
+</footer>
```

Comment:
- [ ] [OPTIONAL] This contact form section is for the next milestone and for this reason I will not perform a review for this section but I would suggest removing it and saving this code for the next milestone so the reviewers can see this code in the Pull Request successfully :raised_hands:

Reply:
Thank you so much my Boss. You are too kind.

Reply:
Hello @alonsofl Please I erroneously merged, please what can I do? I have effected the requested changes

Reply:
The contact form has been removed now. Thank you.